projects/pydentity-wallet.md

```diff
@@ -0,0 +1,58 @@
+# Project Name
+PyDentity Wallet
+
+# Preferred Maturity Level
+Labs stage
+
+# Project Description
+The PyDentity Wallet is a Progressive Web Application for providing [acapy](https://github.com/openwallet-foundation/acapy) access to mobile wallet features and browser API access.
+
+The wallet provides a frontend for a multitenant acapy instance which can be bound to a local mobile device storage.
+
+The app works with iOS and Android and allows users to easily manage an acapy subwallet.
+
+Whenver possible, it uses existing open standards from W3C, IETF, DIF, and other open standards groups.
+
+# Alignment with the OpenWallet Foundation Mission
+This project aligns with the OWF mission, because it's an open source implementation of wallet-related open standards and enhances an existing OWF impact project.
+
+# Code of Conduct
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
+
+# TAC Sponsor
+[Stephen Curran](https://github.com/swcurran)
+
+# Project License
+Apache 2.0
+
+# Source Control
+Currently on a private repository, this project will be a new project at OWF and all existing code will be contributed.
+
+# Issue Tracker
+N/A (not applicable), will track issues through the new GH project.
+
+# External Dependencies
+[Tabler](https://github.com/tabler/tabler) ([MIT](https://github.com/tabler/tabler/blob/dev/LICENSE))
+
+Open Sourced Python Packages, managed with poetry (see pyproject.toml)
+
+Open Sourced Javascript Libraries
+
+# Release Methodology
+Will have semantic versionning, published on gh packages. It can be installed natively from a mobile broswer.
+Possible to release a version on iOS/Android.
+
+# Initial Maintainers
+- [Patrick St-Louis](https://github.com/PatStLouis)
+- [Jamie Hale](https://github.com/jamshale)
+- [Emiliano Suñé](https://github.com/esune)
+- [Lucas O'Neil](https://github.com/loneil)
+
+# Proposed Project Governance
+Follow TAC advice.
+
+# Financial Sponsorship
+None
+
+# Infrastructure
+This will require GitHub Repos with Issues/Bug Tracking, GitHub CI Actions, and mailing list.
```

Comment:
Do we plan to have a live demo? If yes, does GitHub Pages provide a sufficient environment?

Reply:
I have recorded some demo leveraging the wallet, I intend to include a demo in the project for getting started. The demo should ideally cover setting up an instance and interacting with available platforms (such as the BC Wallet Showcase and the VC Playground)